Tevling/Pages/PrivacyPolicy.razor

```diff
@@ -0,0 +1,18 @@
+@page "/PrivacyPolicy"
+@attribute [AllowAnonymous]
+@layout LoginLayout
+@inject IStringLocalizer<PrivacyPolicy> Loc
+
+
+<PageTitle>Tevling - Privacy Policy</PageTitle>
+<div class="privacy-policy p-5">
+    <h1 class="">Tevling - @Loc["PrivacyPolicy"]</h1>
+</div>
```

Comment:
Do you need this div wrapper for the title? A flexbox with a single element inside seems a bit excessive..